src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -247,7 +249,8 @@ def default_api_version(self):
         ResourceType.DATA_COSMOS_TABLE: '2017-04-17',
         ResourceType.MGMT_APPSERVICE: '2018-02-01',
         ResourceType.MGMT_EVENTHUB: '2018-01-01-preview',
-        ResourceType.MGMT_IOTHUB: '2019-07-01-preview'
+        ResourceType.MGMT_IOTHUB: '2019-07-01-preview',
+        ResourceType.MGMT_DATABOXEDGE: '2019-08-01'
```

Comment:
Have we tested this service works in Azure Stack?

Reply:
Yes